src/main/java/com/cbfacademy/frugalflyer/flights/airport/AirportRepository.java

```diff
@@ -0,0 +1,18 @@
+package com.cbfacademy.frugalflyer.flights.airport;
+
+
+
+import org.springframework.data.repository.ListCrudRepository;
+
+
+
+/**
+ * The AirportRepository interface defines the operations for managing Airports in the system.
+ * It provides methods for retrieving Airport records.
+ */
+public interface AirportRepository extends ListCrudRepository<Airport,String> {
+    
+    Airport findByCodeIgnoreCase(String code);
+
+    public void deleteByCodeIgnoreCase(String code);
```

Comment:
```suggestion
    @Modifying
    @Transactional
    public void deleteByCodeIgnoreCase(String code);
```

Custom update/delete operations require both `@Modifying` and `@Transactional`